### DISCUSSION TITLE
There is no "create new database file" button

### DISCUSSION THREAD

**Discussion by MintCat98** [pawelsalawa/sqlitestudio#4788] [Q&A] [answered]
![image](https://github.com/pawelsalawa/sqlitestudio/assets/76907376/9086ecd7-b1d2-41e5-b098-e9633ab9e727)

I just installed sqlitestudio 2 hours ago.
I didn't change settings except for styles, however, there is no "create new database file" button when I click the "Add a database" button.
Do you know how I can fix this?

**Comment by pawelsalawa** [comment, marked as answer]
https://github.com/pawelsalawa/sqlitestudio/discussions/4730
